webpack.config.js

```diff
@@ -37,24 +33,19 @@ const config = [
     module: moduleJS(ENV, PATHS),
     plugins: [
       ...pluginJS(ENV, PATHS),
-      new webpack.IgnorePlugin(/^\.\/locale$/, /moment$/),
-      // Most vendor libs are loaded directly into the 'vendor' bundle (through require()
-      // calls in vendor.js). This ensures that any further require() calls in other
-      // bundles aren't duplicating libs.
-      new webpack.optimize.CommonsChunkPlugin({
-        name: 'vendor',
-        minChunks: module => module.context && module.context.indexOf('/node_modules/') > -1,
+      new webpack.IgnorePlugin({ resourceRegExp: /^\.\/locale$/, contextRegExp: /moment$/ }),
```

Comment:
That bit copies the moment local files to a folder under dist. We should aim to remove moment altogether and use native JS method.